chamberconnectlibrary/p300.py

```diff
@@ -121,7 +119,9 @@ def read_date_time(self):
         returns:
             {"year":int,"month":int,"day":int, "hour":int, "minute":int, "second":int}
         '''
-        date, time = self.ctlr.interact(['DATE?', 'TIME?'])
+        #date, time = self.ctlr.interact(['DATE?', 'TIME?'])
+        date = (self.ctlr.interact('DATE?'))        
```

Comment:
I see in vs code empty spaces at the end. Do you also see them?

Reply:
Yeah I see them too. However, this is part of the code that I didn't touch at all and I would suggest leaving it as is to keep the difference to upstream as small as possible.

Reply:
all good then.